can/bus.py

```diff
@@ -303,6 +339,9 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.shutdown()
 
+    def __del__(self):
+        self.shutdown()
```

Comment:
Not all implementations may be able to handle shutdown being called twice.

Reply:
Fair point - I'd rather remove the `__del__` from the bus instead of adding shutdown state tracking. Do you agree?

Reply:
I think so too.